configurations/ca-staging.ts

```diff
@@ -74,34 +164,135 @@ const translations: Translations = {
     FieldValidationRequiredField: 'Champs requis',
     PreEngagementDescription: 'Commençons',
     BotGreeting: 'Comment je peux aider?',
+
+    // Needs to be translated
+    PreEngagementConfigDescription:
```

Comment:
I think the French translations are linked on the ticket? https://docs.google.com/document/d/1xVUjbhFcDIC-fSlYL7RPeiLsYvL0Ju8p

Reply:
@stephenhand, sorry I missed it. It's done.